atmosphere/settings/__init__.py

```diff
@@ -553,7 +553,7 @@ def generate_secret_key(secret_key_path):
 """
 Import local settings specific to the server, and secrets not checked into Git.
 """
-from atmosphere.settings.local import *
```

Comment:
Oh, good catch.  I think the same would apply to api/__init__.py (file #2 above)?